dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -52,6 +53,8 @@ public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGit
 
     protected List<ProcessingErrors> processImagesWithTag(Namespace ns, List<ImageTagStoreContent> imageNamesWithTag) {
         Integer gitApiSearchLimit = ns.get(Constants.GIT_API_SEARCH_LIMIT);
+        RateLimiter rateLimiter = getRateLimiter();
```

Comment:
We should probably create this as at the `execute` method level and pass this as a parameter to processImageWithTag.  We should create these utilities as close to the entry point of this class as possible.